packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -94,6 +97,30 @@ type ToggleSwitchProps = {
  *   // Other props are applied to input
  *   data-dummy='value' // applied to input
  * />
+ *
+ * @example
+ * <caption>ThemeProvider's consistentPropsSpread=true or wrapperProps != null</caption>
+ * <ThemeProvider future={{ consistentPropsSpread: true }}>
+ *   <ToggleSwitch
+ *     className='my-class' // applied to input
+ *     style={{ padding: 16 }} // applied to input
+ *
+ *     // Other props are applied to input
+ *     data-dummy='value' // applied to input
+ *   />
+ * </ThemeProvider>
+ *
+ * @example
+ * <caption>ThemeProvider's consistentPropsSpread=false/undefined</caption>
+ * <ThemeProvider>
+ *   <ToggleSwitch
+ *     className='my-class' // applied to wrapper
+ *     style={{ padding: 16 }} // applied to wrapper
+ *
+ *     // Other props are applied to input
+ *     data-dummy='value' // applied to input
+ *   />
+ * </ThemeProvider>
```

Comment:
These cases are probably worth explaining in our dedicated docs (`toggleswitch.mdx`), rather than in JSDocs.